airbyte-cdk/python/unit_tests/sources/streams/http/requests_native_auth/test_requests_native_auth.py

```diff
@@ -205,6 +207,61 @@ def test_auth_call_method(self, mocker):
 
         assert {"Authorization": "Bearer access_token"} == prepared_request.headers
 
+    def test_refresh_access_token_invalid(self, requests_mock, mocker):
```

Comment:
what's the difference between this test and the one below? let's extend the below test as it is parametrized, and remove this one please